packages/core-sdk/src/types/resources/nftClient.ts

```diff
@@ -6,14 +6,22 @@ import { EncodedTxData } from "../../abi/generated";
 export type CreateNFTCollectionRequest = {
   name: string;
   symbol: string;
+  /** Whether the minting is public. */
   isPublicMinting: boolean;
+  /** Whether the minting is open. */
   mintOpen: boolean;
+  /** If true, anyone can mint from the collection. If false, only the addresses with the minter role can mint. */
   mintFeeRecipient: Address;
+  /** The contract URI for the collection. Follows ERC-7572 standard. See https://eips.ethereum.org/EIPS/eip-7572. */
   contractURI: string;
+  /** The base URI for the collection. If baseURI is not empty, tokenURI will be either baseURI + token ID (if nftMetadataURI is empty) or baseURI + nftMetadataURI. */
   baseURI?: string;
   maxSupply?: number;
+  /** The cost to mint a token. */
   mintFee?: bigint;
+  /** The token to mint. */
   mintFeeToken?: Hex;
```

Comment:
should this be an an address? let's add this to the next release. no  need to fix in this PR.